geode-core/src/main/java/org/apache/geode/security/AuthInitialize.java

```diff
@@ -92,8 +92,8 @@ Properties getCredentials(Properties securityProps, DistributedMember server, bo
    * @param securityProps
    * @return the credentials to be used. It needs to contain "security-username" and
    *         "security-password"
-   * @deprecated As of Geode 1.3, please implement getCredentials(Properties, DistributedMember,
-   *             boolean)
+   * @deprecated in Geode 1.3. never called by the product. use getCredentials(Properties
```

Comment:
I don't know if the docs side of things differs, but I think two spaces after a period is still the standard for monospaced fonts.  Capitalize after the periods in any case.

Reply:
Also, linking the javadoc would be nice.

```
   * @deprecated in Geode 1.3.  Never called by the product.
   * Use {@link #getCredentials(Properties, DistributedMember, boolean)}
```